server/views/shared/actionPlan.njk

```diff
@@ -15,43 +15,60 @@
 {% block pageContent %}
     {% block content %}
     <div class="govuk-!-width-two-thirds">
-        {{ govukBackLink(backLinkArgs) }}
-        {% if errorSummaryArgs !== null %}
-            {{ govukErrorSummary(errorSummaryArgs) }}
-        {% endif %}
-        <h1 class="govuk-heading-l">View action plan</h1>
+        {% if presenter.probationPractitionerBlockedFromViewing %}
```

Comment:
What do you think about perhaps a controller test, too? Just something to check that message actually appears on the page. I'm in two minds to be honest, and would like to hear your thoughts.

Reply:
yeh - i kinda figured i'd write an integration test to cover it in the future instead (once action plan editing is finished e2e)